src/orders/views/OrderDetails/index.tsx

```diff
@@ -287,7 +307,11 @@ export const OrderDetails: React.FC<OrderDetailsProps> = ({ id, params }) => {
                           onInvoiceSend={id =>
                             openModal("invoice-send", { id })
                           }
-                          onSubmit={handleSubmit}
+                          onSubmit={
+                            order?.status === OrderStatus.UNCONFIRMED
+                              ? handleConfirmationSubmit
+                              : handleSubmit
+                          }
```

Comment:
Can we move this logic to submit handler and decide there whether order should be confirmed or not?